runtime/v2/manager.go

```diff
@@ -314,41 +315,28 @@ func (m *ShimManager) resolveRuntimePath(runtime string) (string, error) {
 		return path.(string), nil
 	}
 
-	var (
-		cmdPath string
-		lerr    error
-	)
-
-	binaryPath := shimbinary.BinaryPath(runtime)
-	if _, serr := os.Stat(binaryPath); serr == nil {
-		cmdPath = binaryPath
-	}
```

Comment:
This pr changes the original logic and I will close the pr

The original logic would first look for shim in the current directory